src/canister/individual_user_template/src/api/hot_or_not_bet/bet_on_currently_viewing_hot_or_not_post.rs

```diff
@@ -49,7 +50,10 @@ async fn bet_on_currently_viewing_post(
         ),
     )
     .await
-    .map_err(|_| BetOnCurrentlyViewingPostError::PostCreatorCanisterCallFailed)?
+    .map_err(|_| {
+        update_token_balance_after_bet_placement_fails(place_bet_arg.clone());
```

Comment:
Should also do this below in case of BettingStatus::BettingClosed ? Although FE wont allow calling but for backend hygiene

Reply:
good call!